plugins/casutg/src/main/java/com/blazemeter/jmeter/threads/AbstractDynamicThreadGroupModel.java

```diff
@@ -24,7 +24,7 @@
     public static final String STEPS = "Steps";
     public static final String ITERATIONS = "Iterations";
     public static final String HOLD = "Hold";
-    protected final Set<DynamicThread> threads = Collections.newSetFromMap(new ConcurrentHashMap<DynamicThread, Boolean>());
+    protected transient Set<DynamicThread> threads = Collections.newSetFromMap(new ConcurrentHashMap<DynamicThread, Boolean>());
```

Comment:
why remove `final`?

Reply:
transient cannot be final  IMU : http://blog.clempinch.com/transient-and-final-instance-variables-in-java/